README.adoc

```diff
@@ -117,18 +107,15 @@ to 18` as a response because the age specified was under the minimum age of 18.
 other combinations of values to get a feel for the constraints that will be defined 
 in this guide.
 
-When you are done, stop the Open Liberty server:
-
-```
-mvn liberty:stop-server
-```
-Now, navigate back to the `start` directory to begin.
+include::{common-includes}/trywhatyoubuild-end.adoc[]
 
 // ==================================================================================
 // Applying constraints on the JavaBeans
 // ==================================================================================
 == Applying constraints on the JavaBeans
 
+Now, navigate back to the `start` directory to begin.
```

Comment:
I think "Navigate to the `start` directory to begin." is good here, as there's a "First" step right after this.

Reply:
Ok, the other issue mentioned it should be moved to match the new format.  Maybe I should change the "first" to a "next"?

Reply:
Making the step to move to the `start` directory to begin as the first main step seems a bit too much. We have the following in our guide's template: 

```
// Write a sentence with the context like "Navigate to the `start` directory to begin." in the section where user starts working with the implementation.
```
I think what makes sense to fit the context is good. As to be consistent with the other guides, moving this sentence from the previous section is the key. But that also make it more difficult to keep one style of sentence that will fit for all guides. This is why we suggest to write this sentence to fit with the context in this section.